### PR TITLE
Oiwfs probe tracking configuration

### PR DESCRIPTION
**Pull request by jluhrs**

### CODE REVIEW

File: modules/model/shared/src/main/scala/navigate/model/package.scala [hunk 1]
```diff
@@ -8,7 +8,7 @@ import cats.Eq
 import java.util.UUID
 
 package model {
```
Comment (cquiroz) [inline modules/model/shared/src/main/scala/navigate/model/package.scala:10]: tiny detail, we could flatten this package structure in scala3

File: modules/server/src/main/scala/navigate/server/tcs/TcsBaseControllerSim.scala [hunk 2]
```diff
@@ -52,4 +52,7 @@ class TcsBaseControllerSim[F[_]: Applicative] extends TcsBaseController[F] {
 
   override def rotIaa(angle: Angle): F[ApplyCommandResult] =
     Applicative[F].pure(ApplyCommandResult.Completed)
+
+  override def oiwfsProbeTracking(config: TrackingConfig): F[ApplyCommandResult] =
+    Applicative[F].pure(ApplyCommandResult.Completed)
```
Comment (cquiroz) [inline modules/server/src/main/scala/navigate/server/tcs/TcsBaseControllerSim.scala:57]: `ApplyCommandResult.Completed.pure[F]` just a comment, dont worry if you prefer the other style

File: modules/server/src/main/scala/navigate/server/tcs/TcsEpicsSystem.scala [hunk 3]
```diff
@@ -293,24 +294,27 @@ object TcsEpicsSystem {
...
+                                rotMoveAngle:     Channel[F, String],
+                                enclosure:        EnclosureChannels[F],
+                                sourceA:          TargetChannels[F],
+                                oiwfs:            TargetChannels[F],
+                                wavelSourceA:     Channel[F, String],
+                                slew:             SlewChannels[F],
+                                rotator:          RotatorChannels[F],
+                                origin:           OriginChannels[F],
+                                focusOffset:      Channel[F, String],
+                                oiProbeTracking:          ProbeTrackingChannels[F]
```
Comment (cquiroz) [inline modules/server/src/main/scala/navigate/server/tcs/TcsEpicsSystem.scala:317]: Missaligned?

File: modules/server/src/main/scala/navigate/server/tcs/TrackingConfig.scala [hunk 4]
```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package navigate.server.tcs
+
+case class TrackingConfig(
+  nodAchopA: Boolean,
```
Comment (cquiroz) [inline modules/server/src/main/scala/navigate/server/tcs/TrackingConfig.scala:7]: Would it help to use opaque types (NewType) to reduce the possibility of flipping parameters?

File: modules/server/src/test/scala/navigate/server/tcs/TcsBaseControllerEpicsSpec.scala [hunk 5]
```diff
@@ -309,37 +322,37 @@ class TcsBaseControllerEpicsSpec extends CatsEffectSuite {
       )
       assert(
         rs.focusOffset.value.exists(x =>
-          compareDouble(x.toDouble, instrumentSpecifics.focusOffset.toMicrometers.value.toDouble)
+          compareDouble(x.toDouble, instrumentSpecifics.focusOffset.toMillimeters.value.toDouble)
```
Comment (cquiroz) [inline modules/server/src/test/scala/navigate/server/tcs/TcsBaseControllerEpicsSpec.scala:325]: uh, good to get the units righ 😄

File: modules/web/server/src/main/scala/navigate/web/server/security/FreeLDAPAuthenticationService.scala [hunk 6]
```diff
@@ -24,11 +24,11 @@ object FreeLDAPAuthenticationService {
   // Operations on ldap
   object LdapOp {
```
Comment (cquiroz) [inline modules/web/server/src/main/scala/navigate/web/server/security/FreeLDAPAuthenticationService.scala:25]: Are we going to use LDAP authentication in navigate?

**Review by cquiroz — APPROVED** [review]
LGTM

File: modules/web/server/src/main/scala/navigate/web/server/security/FreeLDAPAuthenticationService.scala (hunk 6 above)
Comment (jluhrs) [reply modules/web/server/src/main/scala/navigate/web/server/security/FreeLDAPAuthenticationService.scala:25]: Whatever we use for Observe, we use for Navigate

**Pull request merged**